node/variant-alpine.md

```diff
@@ -0,0 +1,9 @@
+## `%%IMAGE%%:<version>-alpine`
+
+This image is based on the popular [Alpine Linux project](https://alpinelinux.org), available in [the `alpine` official image](https://hub.docker.com/_/alpine). Alpine Linux is much smaller than most distribution base images (~5MB), and thus leads to much slimmer images in general.
+
+Please be aware that our Node.js Alpine Docker images use builds from https://unofficial-builds.nodejs.org/, which are not signed by the official Node.js release team. These builds are chosen to ensure compatibility and performance within the Alpine environment. We share this information for transparency and to help you make informed decisions regarding your application setup.
```

Comment:
I think this should mention:

- it's called "unofficial builds", but it is maintained by the node build team
- link to https://github.com/nodejs/node/blob/main/BUILDING.md#platform-list which has musl/alpine as "experimental"